File: 268-missing-number/missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int n = nums.length;
        int sum = n * (n + 1) / 2;
        int array_sum = 0;

        for(int i=0; i<=n-1; i++){
            array_sum += nums[i];
        }
        
        return sum - array_sum;
    }
}

File: 268-missing-number/missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        sum = 0
        res = 0
        n = len(nums)

        for i in range(len(nums)):
            sum = sum + nums[i]
            
        res = n * (n + 1) // 2
            
        return res - sum


        